docs/self-hosted-appcircle/install-server/kubernetes.md

```diff
@@ -0,0 +1,432 @@
+      -----END CERTIFICATE-----
+      -----BEGIN CERTIFICATE-----
+      MIIFazCCA1OgAwIBAgIRAIIQz7DSQONZRGPgu2OCiwAwDQYJKoZIhvcNAQELBQAw
+      ...
+      emyPxgcYxn/eR44/KJ4EBs+lVDR3veyJm+kXQ99b21/+jh5Xos1AnX5iItreGCc=
+      -----END CERTIFICATE-----
+    # Private key for the SSL certificate
+    key: |
+      -----BEGIN PRIVATE KEY-----
+      MIIEvwIBADANBgkqhkiG9w0BAQEFAASCBKkwggSlAgEAAoIBAQC3wS87baGONXjr
+      ...
+      oUcjMAu/mGJjtn9AS0S7rRa58Q==
+      -----END PRIVATE KEY-----
+    # Certificate Authority public key - Typically the bottom certificate of the fullchain SSL certificate
+    caCert: |
+      -----BEGIN CERTIFICATE-----
+      MIIFazCCA1OgAwIBAgIRAIIQz7DSQONZRGPgu2OCiwAwDQYJKoZIhvcNAQELBQAw
+      ...
+      emyPxgcYxn/eR44/KJ4EBs+lVDR3veyJm+kXQ99b21/+jh5Xos1AnX5iItreGCc=
+      -----END CERTIFICATE-----
+# Authentication configuration
+auth:
+  auth-keycloak:
+    # Organization name for Appcircle server
+    organizationName: spacetech
+    # Initial admin user email for Appcircle server
+    initialUsername: "admin@example.com"
+    # Initial admin password - Should contain: min 6 chars, 1 lowercase, 1 uppercase, 1 number
+    # You can create a secret with the password or directly enter the password here
+    initialPassword: "superSecretAppcirclePassword1234"
+    image:
+      # Appcircle keycloak image repository path
+      repository: europe-west1-docker.pkg.dev/appcircle/docker-registry/appcircle-keycloak
+
+# Appcircle vault configuration
+vault:
+  server:
+    image:
+      # Appcircle vault image repository path
+      repository: europe-west1-docker.pkg.dev/appcircle/docker-registry/appcircle-vault
+
+# Web event Redis configuration
+webeventredis:
+  # Enable TLS for Redis connections
+  tls:
+    enabled: true
+  # Ingress configuration for Redis
+  ingress:
+    enabled: true
+    tls: true
+```
+
+## Pre-Installation Steps
+
+### Create Container Registry Secret
+
+By default, Appcircle uses its own image registry that you should authenticate with the `cred.json` file you got from Appcircle.
+
+You can create this secret on the `appcircle` namespace for the Appcircle image registry by executing the following command:
+
+:::tip
+If you are using your own container registry, make sure to update the `server`, `username`, and `password` variables accordingly.
+
+If the registry you are using doesn't require any authentication, you can skip this section.
+:::
+
+1. Create the `appcircle` namespace if you haven't already created it:
+
+:::info
+In this documentation, we will deploy all the resources under the `appcircle` namespace. You can change this to any other namespace if you prefer.
+:::
+
+```bash
+kubectl create namespace appcircle
+```
+
+2. Save the `cred.json` file or your own registry password into a file named `registry-password`.
+
+3. Run the following command on your **Linux/MacOS** terminal to create the container registry secret on the Kubernetes cluster:
```

Comment:
We will mention our docs is compatible with Linux or macOS terminals in the overview section once.